cluster_config.sh.example

```diff
@@ -1,11 +1,53 @@
 eval "$(go env)"
 
-export OPENSHIFT_INSTALL_DATA="$GOPATH/src/github.com/openshift/installer/data/data"
+# To install the latest nightly, run
+#   ./run_ocp.sh
+#
+# To install the latest nightly from 4.9, run
+#   OPENSHIFT_RELEASE=4.9 ./run_ocp.sh
+#
+# To install with a specific release image, run
+#   OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE=<image> ./run_ocp.sh
+#
+# To install the default release of a specific installer binary, run
+#   OPENSHIFT_INSTALLER=/path/to/openshift-install ./run_ocp.sh
+#
+# To install a specific release image with a specific installer binary, run
+#   OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE=<image> \
+#   OPENSHIFT_INSTALLER=/path/to/openshift-install ./run_ocp.sh
+
+scriptdir=$(dirname $0)
+
 export BASE_DOMAIN=shiftstack.test
 
+PULL_SECRET_FILE=${PULL_SECRET_FILE:-"$scriptdir/pull-secret.json"}
+
 # Get your own pull secret from try.openshift.com
-export PULL_SECRET='{"auths": { "quay.io": { "auth": "xxx", "email": "" }}}'
-export SSH_PUB_KEY="`cat $HOME/.ssh/id_rsa.pub`"
+export PULL_SECRET=${PULL_SECRET:-$(jq -c < "$PULL_SECRET_FILE")}
+export SSH_PUB_KEY="$(cat $HOME/.ssh/id_rsa.pub)"
+
+# If OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE is set then use that, otherwise
+# use the latest release from $OPENSHIFT_RELEASE
```

Comment:
maybe one wants to use the release image bundled with the `OPENSHIFT_INSTALLER` they're setting in the env?

Reply:
Ahem: https://github.com/shiftstack/shiftstack-ci/blob/c50c5ee35b745a56958b34a0712d7d20e472514b/cluster_config.sh.example#L12-L13

Reply:
To be fair, this would be clearer if that block was contained within the block below, which is the only scope where it's relevant.

Reply:
oh ok, this `openshift_release_image_override` is only used if `OPENSHIFT_INSTALLER` is not set

https://github.com/shiftstack/shiftstack-ci/pull/173/files#diff-c963d4ad8752a4099e4da7152ce80f85d54ca60d70233ef10833643fe473146aR46

All good then I guess.